src/components/drawer_and_header/user_dropdown.tsx

```diff
@@ -8,10 +8,11 @@ import {
   AnalyticsCategory,
 } from '../../context/analytics_context/analytics_context';
 
-export interface UserDropdownProps {}
+export interface UserDropdownProps { }
 
 export function UserDropdown(props: UserDropdownProps) {
   const authContext = useSession();
+  console.log(authContext)
```

Comment:
intended?

Reply:
Nope, but a commit from a few hours ago (my latest one) should have removed this... Is this on an older commit of the file?

Reply:
stale window maybe?